src/main/java/org/codehaus/mojo/exec/ExecMojo.java

```diff
@@ -478,7 +478,13 @@ public void execute() throws MojoExecutionException {
     private Map<String, String> handleSystemEnvVariables() throws MojoExecutionException {
 
         Map<String, String> enviro = new HashMap<>();
-        Properties systemEnvVars = CommandLineUtils.getSystemEnvVars();
+
+        // Avoid creating env vars that differ only in case on Windows.
+        // https://github.com/mojohaus/exec-maven-plugin/issues/328
+        // It is not enough to avoid duplicates; we must preserve the case found in the "natural" environment.
+        // https://developercommunity.visualstudio.com/t/Build-Error:-MSB6001-in-Maven-Build/10527486?sort=newest
+        Properties systemEnvVars = CommandLineUtils.getSystemEnvVars(true);
```

Comment:
Why not use `System.getenv()` it will be the same

Reply:
I wanted to change as little as possible.  I'm just adding the `true` instead of the default `false`, but if you think `System.getenv()` is better I can change it

Reply:
Please use `System.getenv()` will be clear